prysm/chain.json

```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "prysm",
+  "status": "live",
+  "network_type": "testnet",
```

Comment:
Is this for a testnet? If so, the chain_name, directory name, and location will all need to change.